.changeset/sour-needles-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@etrigan/feature-toggles-ld': minor
```

Comment:
Should this be a major bump?

Reply:
Not sure if this change will affect projects dependent on this lib.

Reply:
Maybe. Would we need to perform a major upgrade to webpack or node (for example) as a result of this? If so, that might be worth a major version bump. Otherwise, a minor bump is fine, as long as there are no breaking changes

Reply:
Maybe. Would we need to perform a major upgrade to webpack or node (for example) as a result of this? If so, that might be worth a major version bump. Otherwise, a minor bump is fine, as long as there are no breaking changes

Reply:
I suppose it could be a major bump... no major code changes were required internally and I don't think much / any will be required upstream, though I'm not really sure tbh. Perhaps a major bump is more appropriate.